definitions/enums/message_type.yml

```diff
@@ -1,5 +1,8 @@
 ---
 name: MessageType
+description: >
+  Let's us differentiate regular messages and
```

Comment:
Maybe "Allows messages to be differentiated between regular messages and system messages."?